### PR TITLE
[#132] rake_task.rb:10-12: Needs to have more parameters to run...

### PR DESCRIPTION
**Pull request by Doldrums**
This is a fix of the puzzle [125-7d80a284](https://github.com/yegor256/pdd/blob/9ff4023b83610ccd06acce934d321ff2edeec35c/lib/pdd/rake_task.rb#L10-L12) from #132 .
The puzzle was created by Alexey Vinogradov on 10-Jul-19.

### CODE REVIEW

**Comment by Doldrums** [comment]
@yegor256 review please

**Comment by AntistasII** [comment]
@yegor256 review this PR please

File: test/test_rake_task.rb [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
-  def test_base
-    PDD::RakeTask.new(:pdd1)
-    error = assert_raises SystemExit do
+  def test_basic
+    Dir.mktmpdir 'test' do |dir|
+      Dir.chdir(dir)
+      File.write('a.txt', "\x40todo #55 hello!")
+      PDD::RakeTask.new(:pdd1) do |task|
+        task.quiet = true
+        # task.license = 'LICENSE.txt'
```
Comment (yegor256) [inline test/test_rake_task.rb:14]: @Doldrums get rid of the comment

File: lib/pdd/rake_task.rb [hunk 2]
```diff
@@ -21,11 +29,11 @@ def initialize(*args, &task_block)
       end
     end
 
+    private
+
     def run
       # @todo #125:30m need to implement this method.
```
Comment (yegor256) [inline lib/pdd/rake_task.rb:35]: @Doldrums I think this line should also be removed

**Comment by yegor256** [comment]
@Doldrums all looks good, just one small comment

**Comment by Doldrums** [comment]
@yegor256 thanks, just deleted it

**Comment by yegor256** [comment]
@rultor merge

**Comment by Doldrums** [comment]
@yegor256 fixed this puzzle description length, actually i didn't get this issue while running `bundle exec rake ` maybe we should add some checks

**Comment by AntistasII** [comment]
@yegor256 Please look at Arina's comment and PR

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**